.github/no-response.yml

```diff
@@ -6,8 +6,5 @@ daysUntilClose: 14
 responseRequiredLabel: "stale"
 # Comment to post when closing an Issue for lack of response. Set to `false` to disable
 closeComment: >
-  This issue has been automatically closed because there has been no response
-  to our request for more information from the original author. With only the
-  information that is currently in the issue, we don't have enough information
-  to take action. Please reach out if you have or find the answers we need so
-  that we can investigate further.
+  This pull request has been automatically closed because there has been no
+  response to our request for information or changes from the original author.
```

Comment:
Maybe add something like "you can comment to re-open" (if that's true) or "you may of course open a new pull request" to not discourage people that missed the deadline?